frontend/src/pages/home/SectionMap.vue

```diff
@@ -802,25 +798,20 @@ watchEffect(() => {
 });
 
 /** auto-adjust right panel/map height */
-const autoRightPanelHeight = ref("");
-const rightPanelBbox = computed(() =>
-  rightPanelElement.value ? useElementBounding(rightPanelElement.value) : null,
+const rightPanelHeight = ref("");
+const columnsTop = computed(
+  () => useElementBounding(columnsElement.value).top.value,
```

Comment:
I'm not sure if this was intended, but this appears to have the side effect of only causing the map to resize to the viewport when the map is vertically repositioned, e.g. when the banner is dismissed. Previously, the map would resize on any change to the size of the viewport.

I don't know if this is wrong, and perhaps there are good reasons for not resizing the map as the viewport changes, e.g. performance and UX, but I noticed it was different than before and didn't see it listed as an explicit change.

Reply:
This was intentional. Since I'm only setting the height on the map with this function, I thought it was appropriate to only trigger that change on vertical changes to the viewport. Its position and width are constrained by css in the typical way. To be more complete, I've also included the window height resizing. Though, in all my observations, actual users almost never resize their windows, they just keep them full screen. Resizing the window horizontally should not re-compute the height now, unless it changes the scroll y position somehow (e.g. browser moves the scroll position to keep something in view, layout shift pushes elements down, etc.).